### PR TITLE
Fix Harvester

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: run/harvest.env [hunk 1]
```diff
@@ -1,8 +1,10 @@
+#!/bin/bash
+
 if [[ -z $STAGE ]]; then
   STAGE=prod
 fi
 
-HARVEST_HOME=$(ruby -ryaml -e "puts YAML::load_file(\"config/settings/$STAGE.yml\")['harvester_home']")
+HARVEST_HOME=$(ruby -ryaml -e "puts YAML::load_file(\"$(dirname $0)/../config/settings/$STAGE.yml\")['harvester_home']")
```
Comment (shelleydoljack) [inline run/harvest.env:8]: I think it'd be better to set an absolute path for the config/settings dir and call it here. Does that not work?

**Review by shelleydoljack — CHANGES_REQUESTED** [review]

File: run/harvest.env (hunk 1 above)
Comment (jgreben) [reply run/harvest.env:8]: It's then just one more place to remember to set the application path.

Comment (jgreben) [reply run/harvest.env:8]: There are already 2 places to set the path. In the settings and in the processor.properties file.

Comment (shelleydoljack) [reply run/harvest.env:8]: ok

**Comment by shelleydoljack** [comment]
Something is still not quite right. The emails for the load, errors, and deactivate users were empty. I see this in the log for last night's run:
```
cat folio-inactive.log.202308302242
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)
/usr/local/rvm/gems/ruby-2.7.1/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.7.1/bin/ruby_executable_hooks:24:in `eval'
/usr/local/rvm/gems/ruby-2.7.1/bin/ruby_executable_hooks:24:in `<main>'
(See full trace by running task with --trace)
```

File: run/do-person-harvest.sh [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
-source harvest.env
+source $(dirname $0)/harvest.env
 
-KEYS=$2
-DATE=$3
+if [[ -n $2 ]]; then
+  KEYS=$2
+fi
+
+if [[ -n $3 ]]; then
```
Comment (jgreben) [inline run/do-person-harvest.sh:9]: This is what was preventing the harvest from running. The date was not being set and and the harvest xml file was coming out as `harvest.xml.out.`

**Review by shelleydoljack — APPROVED** [review]

**Review by shelleydoljack — APPROVED** [review]

**Pull request merged**